src/Tools/ExternalAccess/FSharp/Classification/FSharpClassificationService.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.VisualStudio.LanguageServices.FSharp.Classification
+{
+    [ExportLanguageService(typeof(IClassificationService), LanguageNames.FSharp), Shared]
+    internal class FSharpClassificationService : IClassificationService
+    {
+        private readonly IFSharpClassificationService _delegatee;
+
+        [ImportingConstructor]
+        public FSharpClassificationService([Import(AllowDefault = true)]IFSharpClassificationService classificationService)
```

Comment:
this way, VS still doesn't crash until FSharp move to new API. we can remove allowdefault once they moved.

Reply:
> this way, VS still doesn't crash until FSharp move to new API

VS should not crash. It will simply return `null` for the `IClassifierService` for F# (which is fine because we call `GetLanguageService` and not `GetRequiredLanguageService`).